test/e2e/sdk/publicUrl/test.mjs

```diff
@@ -0,0 +1,19 @@
+import assert from 'node:assert/strict';
+
+import { ApifyClient } from 'apify';
+
+const PUBLIC_DATA = { exposedData: 'test' };
+
+const client = new ApifyClient({
+    token: process.env.APIFY_TOKEN,
+});
+
+const actor = client.actor(process.argv[2]);
+
+const run = await actor.call({ data: PUBLIC_DATA, recordKey: 'public-record-key' }, { waitSecs: 15 });
```

Comment:
(opt) `public-record-key` is actually not needed to be passed as input, it's completely internal :see_no_evil: You will get it as part of the public URL, but you don't care what's in it, you only care that the URL works.

 What you might want to pass in is `urlToPublicData`. But you could also just upload the URL to the dataset and fetch it from there.

Anyway, sorry for all the back and forths on this :sweat_smile: